scintilla/win32/Scintilla.vcxproj

```diff
@@ -91,7 +91,7 @@
   <ItemDefinitionGroup>
     <ClCompile>
       <WarningLevel>Level4</WarningLevel>
-      <PreprocessorDefinitions>WIN32;SCI_EMPTYCATALOGUE;_CRT_SECURE_NO_DEPRECATE;_SCL_SECURE_NO_WARNINGS;_WINDOWS;_USRDLL;_CRT_SECURE_CPP_OVERLOAD_STANDARD_NAMES;SCI_OWNREGEX;SCI_NAMESPACE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>WIN32;SCI_EMPTYCATALOGUE;_CRT_SECURE_NO_DEPRECATE;_SCL_SECURE_NO_WARNINGS;_WINDOWS;_USRDLL;_CRT_SECURE_CPP_OVERLOAD_STANDARD_NAMES;_CRT_SECURE_CPP_OVERLOAD_STANDARD_NAMES_COUNT;SCI_OWNREGEX;SCI_NAMESPACE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
Why not?
But the only concern to me is in the case that if non-security function (for example strcpy) is used but the buffer is allocated dynamically, it won't compile.
And since we are always trying to not modify (minimum) Scintilla project, we won't change it to secured version (for example strcpy_s) in it.

My point is, we don't add such flag into the projects which are not maintained, because we have no control on it.

Reply:
From docu:
The template overloads only work for static arrays. Dynamically allocated buffers require other source code changes.

Currently we use
_CRT_SECURE_NO_WARNINGS
and
_SCL_SECURE_NO_WARNINGS

to silence the related warnings.

Anyhow I will revert that changes.